astropy_helpers/sphinx/conf.py

```diff
@@ -0,0 +1,5 @@
+import warnings
+
+from sphinx_astropy.conf import * 
+
+warnings.warn("Note that astropy_helpers.sphinx.conf is deprecated - use sphinx_astropy.conf instead")
```

Comment:
(minor) maybe "use sphinx_astropy.conf in the sphinx-astropy package"?  Just so a random user who pays no attention this stuff knows what to google...